docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+  crawler:
+    image: docker.elastic.co/integrations/crawler:${CRAWLER_VERSION:-0.2.0}
```

Comment:
When we support `:latest` this should be updated. See https://github.com/elastic/crawler/issues/169